helm/g2p-sandbox/values.yaml

```diff
@@ -325,11 +467,20 @@ ph-ee-engine:
   importer_rdbms:
     enabled: true
     image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-importer-rdbms"
-    imageTag: latest
+    imageTag: "latest"
     LOGGING_LEVEL_ROOT: "DEBUG"
 
   keycloak:
     enabled: true
+    ingress:
+      enabled: true
+      ingressClassName: "kong"
+      rules:
+        - host: 'keycloak.sandbox.fynarfin.io'
```

Comment:
keep this as mifos.io and change from cookbooks

Reply:
Changed